### ISSUE TITLE
There are no items on taskbar (wintc-taskband) except start button, open windows, and clock

### ISSUE THREAD

**Issue by toprak** [rozniak/xfce-winxp-tc#213]
![image](https://github.com/rozniak/xfce-winxp-tc/assets/43301560/fece0a11-0004-42fc-851d-4ed00ce8d1c8)
I did not understand why though, I set wintc-taskband and wintc-desktop as startup applications and changed styles from appearence settings, but there are no volume icon or any other systray icons is it expected?
Using Arch Linux 6.4.8-arch1-1 , xfce-winxp-tc-git r246.cdb952e-1 (from aur)

**Comment by toprak** [comment]
also there were no plugins for xfce panel for start menu or stuff that is shown on that [yt video]( https://youtu.be/wKZqrG9GrdI?t=1638) added time that he shows plugins

**Comment by rozniak** [comment]
>but there are no volume icon or any other systray icons is it expected?

Have a look under the `program-taskband` tag: https://github.com/rozniak/xfce-winxp-tc/issues?q=is%3Aissue+is%3Aopen+label%3Aprogram-taskband

In particular #76 and #77 :stuck_out_tongue: 

>also there were no plugins for xfce panel for start menu or stuff that is shown on that [yt video](https://youtu.be/wKZqrG9GrdI?t=1638) added time that he shows plugins

That is because that video demonstrates and older version of this repo. In the past the Start menu and notification area were implemented as plugins for XFCE panel (taskbuttons didn't even exist iirc at that time). I did some work to basically integrate them into a single program, `wintc-taskband`, to replace the panel.

(This change was done in https://github.com/rozniak/xfce-winxp-tc/commit/2811c5b0714739cb9ff01935a46cee6ed2a8582f )
